analyzers/tests/SonarAnalyzer.UnitTest/TestCases/ConditionalSimplification.FromCSharp10.Fixed.cs

```diff
@@ -9,3 +10,27 @@
 };
 
 class Fruit { public List<int> Prop; }
+
+class FPRepro_5789
```

Comment:
I would not add this example. It only complicates the tests without any additional value. We had a noncompliant FP already that was fixed by this PR and I think it's sufficient.